CodingChallenges/CC_050_1b_CirclePackingAnimatedText_p5_js/index.html

```diff
@@ -1,19 +1,17 @@
 <html>
- <head> 
-  <meta charset="UTF-8" /> 
-  <!-- PLEASE NO CHANGES BELOW THIS LINE (UNTIL I SAY SO) --> 
-  <script language="javascript" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.1/p5.js"></script> 
-  <!-- uncomment lines below to include extra p5 libraries --> 
-  <!--<script language="javascript" src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.1/addons/p5.dom.js"></script>--> 
-  <!--<script language="javascript" src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.1/addons/p5.sound.js"></script>--> 
-  <script language="javascript" type="text/javascript" src="sketch_170111a.js"></script> 
-  <!-- OK, YOU CAN MAKE CHANGES BELOW THIS LINE AGAIN --> 
-  <!-- this line removes any default padding and style. you might only need one of these values set. --> 
-  <style> body {padding: 0; margin: 0;} </style> 
-  <script language="javascript" type="text/javascript" src="sketch.js"></script> 
-  <script language="javascript" type="text/javascript" src="Circle.js"></script> 
-  <script language="javascript" type="text/javascript" src="p5js-temp-Circle993374999066010459.js" class=" temp"></script>
- </head> 
- <body>   
+ <head>
+  <meta charset="UTF-8" />
+  <!-- PLEASE NO CHANGES BELOW THIS LINE (UNTIL I SAY SO) -->
```

Comment:
Where did these comments come from?

Reply:
I dunno, we should probably replace this with the standard one?

Reply:
Yes, I don't think any of these comments are necessary! I'm happy to edit directly if you prefer.